statsmodels/genmod/tests/results/results_glm.py

```diff
@@ -684,7 +684,9 @@ def __init__(self):
         # data set up for data not in datasets
         filename = os.path.join(os.path.dirname(os.path.abspath(__file__)),
             "stata_lbw_glm.csv")
-        data=np.recfromcsv(open(filename, 'rb'), converters={4: lambda s: s.strip(asbytes("\""))})
+        data=np.recfromcsv(open(filename, 'rb'))
```

Comment:
Not sure why, but when you have `converters`, numpy 1.14.0 drops the column. So I think this is an easy workaround. Probably not perfect.